### PR TITLE
Autocomplete Search pf4

### PR DESCRIPTION
**Pull request by DivyanshiSingh**
Fixes: #146 
1) Implemented autocomplete component.
2) Changed search bar position for screens below 768px.
3) Used debounce in search.

### CODE REVIEW

File: src/components/Plugins/Plugins.js [hunk 1]
```diff
@@ -106,9 +112,11 @@ export class Plugins extends Component {
   }
 
   fetchPlugins() {
+    const { location: { search } } = this.props;
     const searchParams = {
       limit: 20,
       offset: 0,
+      name_title_category: search.split('?')[1],
```
Comment (jennydaman) [inline src/components/Plugins/Plugins.js:119]: For type consistency, I would do a

```javascript
search.split('?')[1] || ''
```

**Review by jennydaman — COMMENTED** [review]
This is pretty amazing! I left just a few nitpicks...

preview

File: src/utils/common.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const debounce = (fn, delay) => {
+  let timeout;
```
[the remark this answers is not in the document]
Comment (DivyanshiSingh) [reply src/utils/common.js:10]: used default parameters to do this

File: src/utils/common.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export const debounce = (fn, delay) => {
```
[the remark this answers is not in the document]
Comment (DivyanshiSingh) [reply src/utils/common.js]: Added the comment.

**Review by jennydaman — APPROVED** [review]

**Comment by jennydaman** [comment]
Thanks @DivyanshiSingh this is a cool feature.

**Pull request merged**